<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewImageDialog</class>
 <widget class="QDialog" name="NewImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>146</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Image</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="imageProperties">
     <item row="0" column="0">
      <widget class="QLabel" name="sizeLabel">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="sizeInputs">
       <item>
        <widget class="QSpinBox" name="widthInput">
         <property name="suffix">
          <string>px</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>800</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="heightInput">
         <property name="suffix">
          <string>px</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>600</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="backgroundLabel">
       <property name="text">
        <string>Background</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="backgroundInput">
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="controls">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>controls</sender>
   <signal>accepted()</signal>
   <receiver>NewImageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>controls</sender>
   <signal>rejected()</signal>
   <receiver>NewImageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
